code/map.dm

```diff
@@ -1102,6 +1103,45 @@ var/global/list/mapNames = list(
 		"the Maru primary zone" = list(/area/station/engine/engineering),
 		"the Hammer primary zone" = list(/area/station/security/main))
 
+/datum/map_settings/devtest
+	name = "DEVTEST"
+	display_name = "Developer Lounge & Co"
+	style = "dev zone"
+	arrivals_type = MAP_SPAWN_CRYO
+	dir_fore = NORTH
+
+	parallax_layers = list(
+		/atom/movable/screen/parallax_layer/space_1/west,
+		/atom/movable/screen/parallax_layer/space_2/west,
+		)
+
+	walls = /turf/simulated/wall/auto/supernorn
+	rwalls = /turf/simulated/wall/auto/reinforced/supernorn
+
+	ext_airlocks = /obj/machinery/door/airlock/pyro/external
+	airlock_style = "pyro"
+
+	windows = /obj/window/auto
+	windows_thin = /obj/window/pyro
+	rwindows = /obj/window/auto/reinforced
+	rwindows_thin = /obj/window/reinforced/pyro
+	windows_crystal = /obj/window/auto/crystal
+	windows_rcrystal = /obj/window/auto/crystal/reinforced
+	window_layer_full = COG2_WINDOW_LAYER
+	window_layer_north = GRILLE_LAYER+0.1
+	window_layer_south = FLY_LAYER+1
+
+	escape_dir = EAST
+
+	merchant_left_centcom = /area/shuttle/merchant_shuttle/left_centcom/cogmap
+	merchant_left_station = /area/shuttle/merchant_shuttle/left_station/cogmap
+	merchant_right_centcom = /area/shuttle/merchant_shuttle/right_centcom/cogmap
+	merchant_right_station = /area/shuttle/merchant_shuttle/right_station/cogmap
+
+	valid_nuke_targets = list("super awesome area" = list(/area/devzone),
+		"another super test area" = list(/area/space))
```

Comment:
no

Reply:
ok

Reply:
You seem to misunderstand. This is invalid code